### PR TITLE
New id prefixs

### PR DESCRIPTION
**Pull request by kurtseifried**
So we can add aliases for some of the vendors GSD tracks.

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -158,15 +158,21 @@ The defined database prefixes and their "home" databases are:
...
-| `PYSEC` | The [PyPI vulnerability database](https://github.com/pypa/advisory-db). Serving `<ID>` in the shared format at `https://api.osv.dev/v1/vulns/<ID>` |
-| `RUSTSEC` | [The Rust crates vulnerability database](https://github.com/rustsec/advisory-db). Serving `<ID>` in the shared format at  `https://github.com/RustSec/advisory-db/blob/osv/crates/<ID>.json` |
 | `GSD` | The GSD database. Serving the shared format [here](https://github.com/cloudsecurityalliance/gsd-database). |
-| `GHSA` | The GitHub Security Advisory database. Serving the shared format [here](https://github.com/github/advisory-database). |
 | `LBSEC` | The LoopBack Advisory Database. Serving the shared format [here](https://github.com/loopbackio/security/tree/main/advisories). |
-| `DSA`/`DLA`/`DTSA` | The [Debian Security Advisory](https://www.debian.org/security/) database. Serving the shared format [here](https://storage.googleapis.com/debian-osv/dsa-osv/). |
+| `MFSA` | The [Mozilla Foundation Security Advisory](https://www.mozilla.org/en-US/security/advisories/) database. Serving `ID` in the shared format at `https://www.mozilla.org/en-US/security/advisories/<ID>/`. |
```
Comment (oliverchang) [inline docs/schema.md:167]: These are meant to specify places to find the OSV formatted entries, which doesn't seem to be the case for most of these?

Comment (kurtseifried) [reply docs/schema.md:167]: I thought this was for specifying ID's and effectively namespacing them, so e.g. if I see "DSA-foo" I don't have to guess (is it debian? is it dell? https://www.dell.com/support/security/en-ca [DSA-2023-129: Dell PowerStore Family Security Update for Multiple Vulnerabilities](https://www.dell.com/support/kbdoc/en-ca/000212002/dsa-2023-129-dell-powerstore-family-security-update-for-multiple-vulnerabilities) and so on. 

is this only for OSV formatted providers?

Comment (oliverchang) [reply docs/schema.md:167]: Indeed, this list is meant to enumerate IDs for OSV-formatted databases.

**Comment by oliverchang** [comment]
Closing per https://github.com/ossf/osv-schema/pull/148#discussion_r1163502463

**Pull request closed without merging**